parallel-install/pkg/components/components_test.go

```diff
@@ -65,3 +65,48 @@ func Test_GetComponents(t *testing.T) {
 	require.Equal(t, 2, len(res), "Number of components not as expected")
 	require.Equal(t, "comp2", res[0].Name)
 }
+
+func Test_ReuseHelmValues(t *testing.T) {
+	// fake k8s with override ConfigMaps
+	k8sMock := fake.NewSimpleClientset(
+		&v1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "global-overrides",
+				Namespace: "kyma-installer",
+				Labels:    map[string]string{"installer": "overrides"},
+			},
+			Data: map[string]string{
+				"globalOverride1": "test1",
+				"globalOverride2": "test2",
+			},
+		},
+	)
+
+	overridesProvider, err := overrides.New(k8sMock, make(map[string]interface{}), logger.NewLogger(true))
+	require.NoError(t, err)
+
+	instCfg := &config.Config{
+		ComponentList: &config.ComponentList{
+			Components: []config.ComponentDefinition{
+				{
```

Comment:
I think you can provide an empty list here as these values are not checked later on (just for simplifying the test).